nbconvert/preprocessors/clearmetadata.py

```diff
@@ -0,0 +1,22 @@
+"""Module containing a preprocessor that removes metadata from code cells"""
+
+# Copyright (c) IPython Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from traitlets import Set
+from .base import Preprocessor
+
+class ClearMetadataPreprocessor(Preprocessor):
+    """
+    Removes the output from all code cells in a notebook.
+    """
+
+    def preprocess_cell(self, cell, resources, cell_index):
+        """
+        Apply a transformation on each cell. See base.py for details.
```

Comment:
This docstring is not specific to the purpose you are aiming at here, could you make it more specific.